Generated API change for kubernetes 1.10

roycaihw opened the pull request:
Generated APIs, models and docs.

`kubernetes/client/` contains the generated APIs and models
`kubernetes/docs/` contains the generated documentation
`kubernetes/README.md` is the generated README


------------

To re-generate, follow the instruction in [client-gen](https://github.com/kubernetes-client/gen#generating-a-client):
`GEN_DIR/openapi/go.sh OUTPUT_DIR SETTING_FILE`

Setting being used:
    `export KUBERNETES_BRANCH="release-1.10"`
    `export CLIENT_VERSION="0.1.0a1"`
    `export PACKAGE_NAME="client"`

The generated `README.md` and `docs/` are manually moved for better arrangement. 

cc @yliaog

yliaog commented:
why is CLIENT_VERSION set to "1.0.0b1"?

https://github.com/roycaihw/go/tree/38e997d8fd30c0408e12b839844720515ccc3ffa/kubernetes/client
shows "Generate go client", it is kind of confusing. It is better to be more descriptive like that for python 
https://github.com/kubernetes-client/python/tree/master/kubernetes/client 
"Generated API change for kubernetes 1.10"

roycaihw committed:
Generated API change for kubernetes 1.10

Settings:
    export KUBERNETES_BRANCH="release-1.10"
    export CLIENT_VERSION="0.1.0a1"
    export PACKAGE_NAME="client"

roycaihw commented:
@yliaog That makes sense. Updated CLIENT_VERSION to 0.1.0a1

yliaog commented:
/lgtm

yliaog reviewed (approved):
lgtm
